autarchy/autarchy.py

```diff
@@ -1,8 +1,82 @@
 """ Main .py for running autarchy """
 from __future__ import print_function
 
+import numpy as np
+from sklearn import model_selection
 import sys
-import tpot
+from tpot import TPOTRegressor
 
-print(len(sys.argv))
+assert len(sys.argv) > 1
```

Comment:
You may want to move `argparse` for reading in arguments. Also, when this fails it will give an `AssertionError`, which isn't super helpful. Perhaps catch and rethrow something better like `raise ValueError("missing input file argument")`. (`ValueError` might not be the right one, but you get the point)